components/Carousel/Carousel.js

```diff
@@ -1,8 +1,13 @@
 class Carousel {
-
+	constructor(element) {
+		this.element = element;
+		this.images = document.querySelectorAll('img');
```

Comment:
Think about where you want to select the `img` element from. `querySelectorAll` is a method for document but also parent nodes.